bats/fb-test-backup.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+BACKUP_DIR=/var/tmp/foreman-backup
```

Comment:
Would it make sense to create a temporary directory?
```suggestion
BACKUP_DIR=$(mktemp -d --tmpdir=/var/tmp/foreman-backup)
```

Reply:
What would be the benefit? The test can be run multiple times?

I've added the `--preserve-directory` so that FM doesn't create it's timestamped dir so the tests don't need to *find* the result. That problem wouldn't exist with `mktemp`, but I am not convinced we gain anything (but less readable paths)

Reply:
Running it multiple times indeed. We generally try to keep our test suite in a way that you can rerun it. Another would be to remove the directory if already present.

Reply:
I'd go with the removal before run then.